### DISCUSSION TITLE
【keyball61】ビルドガイドの3-8. LED（オプション）のはんだ付け をしてから左右のPro Microが認識されない

### DISCUSSION THREAD

**Discussion by hyumiretto** [Yowkees/keyball#652] [Q&A]
タイトルの問題が発生しました。

【問題発生前まで】
3-7で左右のProMicro書き込みと全キーの確認まで終わりました（左トラックボールで実装です。

※5左手側の左右判定ジャンパーをはんだ付けのみ先行して実装しています。

【問題発生後】
3-8でLEDを左右ともに実装後、ProMicroをType-Cで繋いだところ両方とも認識されなくなりました（LEDの実装中には一度もType-Cで繋いでいません。

ご助力いただけると、嬉しく思います。

下記、問題発生後の画像です。

![IMG_6738](https://github.com/user-attachments/assets/4be65901-a07a-4efd-8fba-e668386fbc14)
![IMG_6737](https://github.com/user-attachments/assets/93356c9f-14f4-48b8-b7ce-b05815b8dea0)

![IMG_6739](https://github.com/user-attachments/assets/54f3d0e2-25fb-47f4-a3a4-db73a8d8a444)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

おそらくLEDの足が隣とショートしてしまっている、もしくはショートモードで故障しているLEDがあると思います。
基板が古くて位置が少々違いますが、下記ディスカッションの回答の通り、問題を切り分けてみてください。
https://github.com/Yowkees/keyball/discussions/407

以上、お手数ですがご対応お願いいたします。

**Comment by hyumiretto** [comment]
すべてのLEDを取り除くと認識されましたので、一旦このまま使用します。

ご回答ありがとうございます。
